src/components/event/MapModal/Map-modal.js

```diff
@@ -1,58 +1,54 @@
-﻿import React, { Component } from 'react';
+﻿import React, { useState } from 'react';
 import Button from '@material-ui/core/Button';
 import Dialog from '@material-ui/core/Dialog';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogTitle from '@material-ui/core/DialogTitle';
-import { LocationMapWithCircle } from '../../helpers/form-helpers/location';
 import { Field } from 'redux-form';
+
+import { LocationMapWithCircle } from '../../helpers/form-helpers/location';
 import '../slider.css';
 import DisplayMap from '../map/display-map';
+import constants from '../../../constants/mapModal';
 
-class MapModal extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            open: false
-        }
-    }
+export const MapModal = (initialize,values) => {
```

Comment:
There is only one parameter in functional component(`props`). Use destruct operator in get these values: `{initialize,values}`